Tests/Pcap++Test/main.cpp

```diff
@@ -1067,6 +1069,81 @@ PTF_TEST_CASE(TestPcapNgFileReadWrite)
 
 }
 
+PTF_TEST_CASE(TestPcapNgFileReadWriteCompress)

```

Comment:
instead of duplicating the existing test-cases, maybe we can add another writer to the original test-case that will use compression and then check the two writers behave the same

Reply:
Since this is optional to build I think it may be wise to keep the test isolated? Otherwise you will have to interweave some pragmas into the existing test. I could still edit the new test to run dual writers and compare results.

Let me know what you think.

Reply:
I just realized what I said here is not entirely true, since it falls back to NON compression mode when zstd isn't built you don't need jumper out any code. 

Anyways if you still feel strongly about re-working this I can do that. Just let me know.

Reply:
Yes, I think it'd be better to integrate this into the existing tests. If zstd is not enabled it will fall back to NON compression

Reply:
Unified